governance.md

```diff
@@ -1,98 +1,137 @@
-# Tekton Bootstrap Governance
+# Tekton Governance
 
-The initial bootstrap committee will consist of 5 individuals and one governance facilitator who are core stakeholders and/or contributors.
+The Tekton Governance committee is the governing body of the Tekton open
+source project. It's an elected group that represents the contributors to
+the project, and has an oversight on governance and technical matters.
 
-Members are:
+## Tekton Governance Committee
 
-* [Andrew Bayer](https://github.com/abayer) (CloudBees)
-* [Vincent Demeester](https://github.com/vdemeester) (Red Hat)
-* [Christie Wilson](https://github.com/bobcatfish) (Google)
-* [Andrea Frittoli](https://github.com/afrittoli) (IBM)
-* [Jason Hall](https://github.com/ImJasonH) (Google)
+The Tekton governance committee consists of five elected individuals and one
+optional facilitator. The five seats are held for two year terms, staggered by
+one year: every year either two or three of the seats are up for election.
+
+### Current members
+
+Full Name         | Company   | GitHub  | Slack         | Elected On
+------------------|:---------:|---------|---------------|-----------
+Andrew Bayer      | CloudBees | [abayer](https://github.com/abayer)         | [@Andrew Bayer](https://tektoncd.slack.com/team/UJ6DJ4MSS)    | Bootstrap Committee
```

Comment:
andrew was elected in the feb 2020 election as well

Reply:
upz, of course, good catch